documents/manifesto.md

```diff
@@ -0,0 +1,28 @@
+# The Lua Users Foundation Manifesto
+
+The Lua Users Foundation (the Foundation) is an association of individuals in a modular umbrella structure consisted of several workgroups, detailed in the Lua Users Foundation Constitution.
+The Foundation formal decision-making process is governed by the Lua Users Foundation Constitution, according to the Foundation Goals and Values, towards our Mission.
+
+## The Mission
+
+  * Support and promote the Lua programming language, its community and ecosystems.
+    
+## The Values
+
+  * The Lua programming language is developed by the Lua Team and defined by the Lua Programming Manual. Furthermore:
+  * The Lua programming language is small, fast, easy to learn and use. 
```

Comment:
Not sure if this line and the previous one make sense here. They are facts not values, and concern the Lua language not the Foundation.

Reply:
This line is there to state our position in the the "is LuaJIT Lua?" controversy

Reply:
OK, it makes sense.